### PR TITLE
Sn1 128 utilize entire page for summarization context

### PR DESCRIPTION
**Pull request by bkb2135**
Use the entire page for summarization

### CODE REVIEW

File: prompting/tools/datasets/wiki.py [hunk 1]
```diff
@@ -208,13 +208,18 @@ def get(
...
         if not sections:
             return None
-
-        key = header, section_title = selector(list(sections.keys()))
-        content = "\n".join(sections[key])
-        section_length = len(content.split())
+        if selector == 'all':
+            content = "\n".join(["\n".join(section) for section in sections.values()])
+            section_length = len(content.split())
+            topic = "All Sections"
```
Comment (dbobrenko) [inline prompting/tools/datasets/wiki.py:214]: Minor suggestion to be consistent with the quotes style across the project, in our case `"`

File: prompting/conversation.py [hunk 2]
```diff
@@ -42,7 +42,10 @@ def create_task(
     if dataset is None:
         raise ValueError(f"Dataset {dataset_name} not found")
     else:
-        dataset = dataset()
+        if task_name == SummarizationTask.name:
+            dataset = dataset(selector='all')
```
Comment (dbobrenko) [inline prompting/conversation.py:46]: Should we just modify the default argument of `WikiDataset.get` for `selector="all"`, to be the code less coupled?

**Review by dbobrenko — APPROVED** [review]
LGTM with minor suggestions

File: prompting/tools/datasets/wiki.py [hunk 3]
```diff
@@ -208,13 +208,18 @@ def get(
         )
         if not sections:
             return None
-
-        key = header, section_title = selector(list(sections.keys()))
-        content = "\n".join(sections[key])
-        section_length = len(content.split())
+        if selector == 'all':
```
Comment (dbobrenko) [inline prompting/tools/datasets/wiki.py:211]:
```suggestion
        if selector == "all":
```
Minor suggestion to be consistent with the quotes style across the project, in our case `"`

File: prompting/conversation.py (hunk 2 above)
Comment (dbobrenko) [inline prompting/conversation.py:46]:
```suggestion
            dataset = dataset(selector="all")
```
Should we just modify the default argument of `WikiDataset.get` for `selector="all"`, to make the code less coupled?

**Comment by bkb2135** [comment]
Replaced by https://github.com/macrocosm-os/prompting/pull/306

**Pull request closed without merging**